code/modules/surgery/surgery_step.dm

```diff
@@ -80,32 +80,41 @@
 		implement_speed_mod = implements[implement_type] / 100.0
 
 	speed_mod /= (get_location_modifier(target) * (1 + surgery.speed_modifier) * implement_speed_mod) * target.mob_surgery_speed_mod
-	var/modded_time = time * speed_mod
+	var/modded_time = time * speed_mod * 2 //SKYRAT EDIT - all surgery times multiplied by 2
 
-
-	fail_prob = min(max(0, modded_time - (time * SURGERY_SLOWDOWN_CAP_MULTIPLIER)),99)//if modded_time > time * modifier, then fail_prob = modded_time - time*modifier. starts at 0, caps at 99
+	// SKYRAT REMOVAL //fail_prob = min(max(0, modded_time - (time * SURGERY_SLOWDOWN_CAP_MULTIPLIER)),99)//if modded_time > time * modifier, then fail_prob = modded_time - time*modifier. starts at 0, caps at 99
 	modded_time = min(modded_time, time * SURGERY_SLOWDOWN_CAP_MULTIPLIER)//also if that, then cap modded_time at time*modifier
 
 	var/was_sleeping = (target.stat != DEAD && target.IsSleeping())
 
-	// Skyrat Edit Addition - reward for doing surgery on calm patients, and for using surgery rooms(ie. surgerying alone)
-	if(was_sleeping || HAS_TRAIT(target, TRAIT_NUMBED) || target.stat == DEAD)
-		modded_time *= SURGERY_SPEEDUP_AREA
-		to_chat(user, span_notice("You are able to work faster due to the patient's calm attitude!"))
-	var/quiet_enviromnent = TRUE
-	for(var/mob/living/carbon/human/loud_people in view(3, target))
-		if(loud_people != user && loud_people != target)
-			quiet_enviromnent = FALSE
-			break
-	if(quiet_enviromnent)
-		modded_time *= SURGERY_SPEEDUP_AREA
-		to_chat(user, span_notice("You are able to work faster due to the quiet environment!"))
-	// Skyrat Edit End
-	// Skyrat Edit: Cyborgs are no longer immune to surgery speedups.
-	//if(iscyborg(user))//any immunities to surgery slowdown should go in this check.
-		//modded_time = time
-	// Skyrat Edit End
```

Comment:
Bear in mind that SURGERY_SPEEDUP_AREA is actually a poorly named variable that was repurposed, and isn't a bonus for using a surgery room as your PR description suggests.

Before your PR, you do not get a bonus to speed for using surgery rooms. You get a bonus for the patient being anesthetized, and another bonus for having no other people within three tiles of the patient, with the intention to reward not penalize the player for proper use of anesthesia, and for not having multiple doctors speedrunning surgeries on a patient.

I like the idea of shock, but I dislike removing the speed buffs. The situational buffs were always noticeable, even if I believe they were overtuned. I don't think both removing these buffs, as well as doubling surgery time is wise. One or the other could easily suffice, along with the introduction of shock.